.github/workflows/appengine-deploy.yml

```diff
@@ -0,0 +1,47 @@
+name: App Engine Deploy Integration
+
+on: 
+  push:
+    branches:
+      - main 
+  pull_request:
```

Comment:
If there's nothing listed under here, is this necessary?

Reply:
It means that pull requests open on any branch should be checked! :)